sass/_aesop.scss

```diff
@@ -100,22 +100,34 @@
 	}
 }
 
-.aesop-stacked-img {
-	background-attachment: fixed;
-	background-position: 50%;
-	background-repeat: no-repeat;
-	background-size: cover;
-	height: 500px; // fallback for browsers that don't support vh
-	height: 100vh;
-    left: 0;
-    overflow: hidden;
-    position: relative;
-    top: 0;
-    width: 100%;
-	
-    @media screen and (max-width: $width-tablet) {
-        background-attachment: scroll;
-        background-size: contain !important; // overrides inline style set by js
+.aesop-stacked {
+    &-img {
+        background-attachment: fixed;
+        background-position: 50%;
+        background-repeat: no-repeat;
+        background-size: cover;
+        height: 500px; // fallback for browsers that don't support vh
+        height: 100vh;
+        left: 0;
+        overflow: hidden;
+        position: relative;
+        top: 0;
+        width: 100%;
+
+        @media screen and (max-width: $width-tablet) {
+            background-attachment: scroll;
+            background-size: contain !important; // overrides inline style set by js
+        }
+    }
+    
+    &-caption {
+        background-image: linear-gradient(rgba(0, 0, 0, .5), rgba(0, 0, 0, 0));
+        color: #fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.